frontend/src/slices/auth/auth.slice.ts

```diff
@@ -2,14 +2,19 @@ import { createSlice } from '@reduxjs/toolkit';
 
 import { DataStatus } from '#libs/enums/enums.js';
 import { type ValueOf } from '#libs/types/types.js';
+import { type UserGetAllItemResponseDto } from '#packages/users/users.js';
 
-import { signUp } from './actions.js';
+import { getAuthenticatedUser, signUp } from './actions.js';
 
 type State = {
+  user: UserGetAllItemResponseDto | null;
+  authenticatedUserDataStatus: ValueOf<typeof DataStatus>;
   dataStatus: ValueOf<typeof DataStatus>;
 };
 
 const initialState: State = {
+  user: null,
```

Comment:
I was thinking about it, and let's rename user to authenticatedUser too, so we preserve consistency, and in case we have other users (like for forget password  we will need a temp user) it will be more obvious